src/classes/info.py

```diff
@@ -47,30 +50,38 @@
 EXPORT_PRESETS_PATH = os.path.join(PATH, "presets")
 
 # User paths
-HOME_PATH = os.path.join(os.path.expanduser("~"))
-USER_PATH = os.path.join(HOME_PATH, ".openshot_qt")
-BACKUP_PATH = os.path.join(USER_PATH)
-RECOVERY_PATH = os.path.join(USER_PATH, "recovery")
-THUMBNAIL_PATH = os.path.join(USER_PATH, "thumbnail")
-CACHE_PATH = os.path.join(USER_PATH, "cache")
-BLENDER_PATH = os.path.join(USER_PATH, "blender")
-TITLE_PATH = os.path.join(USER_PATH, "title")
-TRANSITIONS_PATH = os.path.join(USER_PATH, "transitions")
-EMOJIS_PATH = os.path.join(USER_PATH, "emojis")
-PREVIEW_CACHE_PATH = os.path.join(USER_PATH, "preview-cache")
-USER_PROFILES_PATH = os.path.join(USER_PATH, "profiles")
-USER_PRESETS_PATH = os.path.join(USER_PATH, "presets")
-USER_TITLES_PATH = os.path.join(USER_PATH, "title_templates")
-PROTOBUF_DATA_PATH = os.path.join(USER_PATH, "protobuf_data")
-YOLO_PATH = os.path.join(USER_PATH, "yolo")
+OLD_HOME_PATH = os.path.join(os.path.expanduser("~"))
+OLD_USER_PATH = os.path.join(OLD_HOME_PATH, ".openshot_qt")
+
+HOME_PATH = QStandardPaths.writableLocation(QStandardPaths.HomeLocation)
+USER_PATH = QStandardPaths.writableLocation(QStandardPaths.DataLocation)
+APP_PATH  = os.path.join(USER_PATH, NAME)
+
+if (os.path.exists(OLD_USER_PATH)) :
+    shutil.move(OLD_USER_PATH, APP_PATH)
```

Comment:
We should copy, not move, to preserve backwards compatibility. I'd rather leave extra files lying around than break people's installs if they switch back to an older version.

Oh! And also because existing user projects may contain pathnames that point into the old directory location. (For situations like Windows projects that were saved to a different drive, etc... We can't rely on the assumption that all paths follow our imperfect conventions, because they are imperfect.)

Reply:
(So, I'd prefer something more like...)
```suggestion
sentry_file = os.path.join(OLD_USER_PATH, ".settingsMigrated")
if (os.path.exists(OLD_USER_PATH) and not os.path.exists(sentry_file)):
    try:
        # Make sure we have write permission, first
        os.makedirs(APP_PATH, exist_ok=True)
        shutil.copytree(
            OLD_USER_PATH,
            APP_PATH,
            dirs_exist_ok=True,
            ignore_dangling_symlinks=True)
        os.mknod(sentry_file)
    except (PermissionError, FileNotFoundError) as ex:
        ERROR_MOVING_CONFIGS = ex
```
...And we'll deal with showing a message or something, when the gui launches.

Reply:
(Hmm. Should the sentry file be located in the _old_ config dir, or the _new_ config dir? I'm kind of conflicted on that one.

I think probably the old dir. Then we can tell users, "Delete the file `.openshot_qt/.settingsMigrated` and OpenShot will re-import that configuration on its next launch (overwriting any settings changes in the new location)."